src/dsp_tools/utils/xmlupload/upload_stashed_resptr_props.py

```diff
@@ -97,6 +74,125 @@ def upload_stashed_resptr_props(
     return nonapplied_resptr_props
 
 
+def _upload_all_resptr_props_of_single_resource(
+    resource_in_triplestore: dict[str, Any],
+    link_prop: XMLProperty,
+    res_iri: str,
+    stashed_resource: XMLResource,
+    resptrs: list[str],
+    id2iri_mapping: dict[str, str],
+    con: Connection,
+    nonapplied_resptr_props: dict[XMLResource, dict[XMLProperty, list[str]]],
+    verbose: bool,
```

Comment:
I suggest reordering the parameters to make it better understandable:
```suggestion
    resource_in_triplestore: dict[str, Any],
    stashed_resource: XMLResource,
    res_iri: str,
    link_prop: XMLProperty,
    resptrs: list[str],
    id2iri_mapping: dict[str, str],
    con: Connection,
    nonapplied_resptr_props: dict[XMLResource, dict[XMLProperty, list[str]]],
    verbose: bool,
```

Reply:
Done